packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -906,19 +906,18 @@ class _DayPickerState extends State<_DayPicker> {
   ///
   /// Examples:
   ///
-  /// ```
-  /// ┌ Sunday is the first day of week in the US (en_US)
-  /// |
-  /// S M T W T F S  <-- the returned list contains these widgets
-  /// _ _ _ _ _ 1 2
-  /// 3 4 5 6 7 8 9
+  ///     ┌ Sunday is the first day of week in the US (en_US)
+  ///     |
+  ///     S M T W T F S  ← the returned list contains these widgets
+  ///     _ _ _ _ _ 1 2
+  ///     3 4 5 6 7 8 9
+  ///
```

Comment:
I would expect that the alignment here will be off when these are rendered with a variable-width font on https://api.flutter.dev/?

![Screen Shot 2022-08-01 at 5 10 15 PM](https://user-images.githubusercontent.com/1227763/182265397-772d45f9-65e1-4bef-bdb5-21d92a94db52.png)

(or does dartdoc have a special feature that I am not aware off and these will continue to render with a fixed-width font even without the ```?)

Reply:
I have the same concern for many other places in this PR...

Reply:
The four space indent triggers monospace font.